File: seed/csharp-model/trace/src/Client/V2/Problem/GetBasicSolutionFileRequest.cs
using System.Text.Json.Serialization;
using Client.V2;

namespace Client.V2;

public class GetBasicSolutionFileRequest
{
    [JsonPropertyName("methodName")]
    public string MethodName { get; init; }

    [JsonPropertyName("signature")]
    public NonVoidFunctionSignature Signature { get; init; }
}

File: seed/rust-sdk/trace/src/types/get_basic_solution_file_request.rs
use crate::non_void_function_signature::NonVoidFunctionSignature;
use serde::{Deserialize, Serialize};

#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
pub struct GetBasicSolutionFileRequest {
    #[serde(rename = "methodName")]
    pub method_name: String,
    pub signature: NonVoidFunctionSignature,
}